test/topology_custom/test_replace_with_encryption.py

```diff
@@ -7,42 +7,54 @@
 from test.pylib.manager_client import ManagerClient
 from test.pylib.scylla_cluster import ReplaceConfig
 
+
 @pytest.mark.asyncio
-async def test_replace_with_encryption(manager: ManagerClient):
+@pytest.mark.parametrize("join_ring", [True, False])
+async def test_replace_with_encryption(manager: ManagerClient, join_ring):
     """Test that a node can be replaced if inter-dc encryption is enabled.
        The test creates 6 node cluster with two DCs and replaces one node in
```

Comment:
Now it creates a 8 node cluster.

Which is pretty large for such simple test, (which has a downside: slower test, more resource consumption in CI etc.) and not sure if so many nodes are needed? Maybe you could reduce the numbers in the initial two `servers_add` calls from 3 to 2, to bring it back down to 6 nodes total again

Reply:
Done